75x Fireworks: hide PF user code by the default

alja opened the pull request:
This is forward port of  #10464.

alja committed:
Set purpose of all Colin's proxy builder starting with 'PF '.

alja committed:
Add option to hide PF user plugins.

alja committed:
Rename PF command line option

alja committed:
Change purpose from 'PF PatJet' to 'PatJet PFConstituents'.

alja committed:
Don't show PF specific views in menu bar when PF is not specifically enabled.

alja committed:
Undo changes from the previous commit.

alja committed:
Hide user detail views if cmsShow option is not available.

alja committed:
Rename command line option --enable-pflow to --expert

alja commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_7_5_X IBs once checked with relvals in the development release cycle of CMSSW or unless it breaks tests. This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @degano, @smuzaffar

alja committed:
Fixes for standalone version.

Conflicts:
	Fireworks/Core/scripts/cmsShow
	Fireworks/Core/standalone_build/make_tarball.sh

alja commented:
+1

lgray commented:
@alja @davidlange6 @slava77 This PR is causing clang build errors in 76X and 75X.

See here: https://github.com/cms-sw/cmssw/pull/10620#issuecomment-129790381

Could someone please fix this? (and perhaps next time run standard tests before merging the PR too)
